packages/core/scope.ts

```diff
@@ -92,8 +92,12 @@ export class Scope {
           if (interaction.interactable === this) {
             interaction.stop()
           }
+          scope.interactions.signals.fire('unset', { interaction })
```

Comment:
Can you rename this from `unset` to `destroy` to match the method?

Reply:
I think I was wrong calling the method `destroy`, because you never use that word in the whole project and use `unset` instead. So I think I will rather change my `destroy` method name to `unset`, it's probably more consistent this way.